mappings/net/minecraft/text/Text.mapping

```diff
@@ -42,6 +42,10 @@ CLASS net/minecraft/class_2561 net/minecraft/text/Text
 		COMMENT <p>A shallow copy is made for the siblings.</p>
 	METHOD method_27662 copy ()Lnet/minecraft/class_5250;
 		COMMENT Copies the text itself, excluding the styles or siblings.
+	METHOD method_30163 fromString (Ljava/lang/String;)Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	METHOD method_30163 of (Ljava/lang/String;)Lnet/minecraft/class_2561;
```
or
```suggestion
	METHOD method_30163 literal (Ljava/lang/String;)Lnet/minecraft/class_2561;
```

Reply:
I went with `of`, since `literal` would imply to me that null's aren't handled.